src/logo.tsx

```diff
@@ -151,10 +152,11 @@ const VivoLogoImage = ({size, type}: LogoImageProps) => {
 };
 
 const O2LogoImage = ({size, type}: LogoImageProps) => {
-    const {isDarkMode} = useTheme();
+    const {isDarkMode, skinName} = useTheme();
     const isInverse = useIsInverseVariant();
     // todo WEB-761 what about classic?
-    const {colors} = getO2Skin();
+    const {colors} = skinName === 'O2-new' ? getO2NewSkin() : getO2Skin();
```

Comment:
Just a side note: with this approach, we won't be able to render o2 new logo when the theme's skin is different from O2-new

Reply:
but the logo (as a form) is the same, why or when would you need to use the old color in the logo? 🤔

Reply:
If you want to render the O2 logo in any other skin different from O2_new, the colors will always be the old ones if we use your implementation.

Reply:
Agree with @marcoskolodny, better create two different components

Reply:
After talking in teams, current implementation is ok to me. When using the logo from a different brand, the "official" o2 logo will be used

Reply:
Finally, I include this change because is need it for Preview Tools to avoid the O2 logo change to new colors for "old skin". You always are right guys! 😁